src/sentry/sentry_metrics/indexer/postgres/postgres_v2.py

```diff
@@ -409,6 +409,22 @@ def reverse_resolve(self, use_case_id: UseCaseID, org_id: int, id: int) -> Optio
         string: str = obj.string
         return string
 
+    def bulk_reverse_resolve(
+        self, use_case_id: UseCaseID, org_id: int, ids: Collection[int]
+    ) -> Mapping[int, str]:
+        ret_val: Dict[int, str] = {}
+        metric_path_key = METRIC_PATH_MAPPING[use_case_id]
+        table = self._get_table_from_metric_path_key(metric_path_key)
+        try:
+            strings = table.objects.using_replica().filter(
```

Comment:
Why? This now works completely differently from reverse_resolve and has different consistency guarantees.

Reply:
Hey, thanks for the input.
I saw that `resolve` uses replica so I thought that would be a similar case.
I'll remove it.

Reply:
I believe resolve is on the other hand explicitly cached in `cache.py`